### PR TITLE
[New Effect] Kifflom!

### PR DESCRIPTION
**Pull request by EliRickard**
All Peds while continuously repeat the name of the Kifflom cult, utilises existing ped speech controller.

### CODE REVIEW

File: ChaosMod/Effects/EffectsInfo.h [hunk 1]
```diff
@@ -257,6 +257,8 @@ enum EffectType
 	EFFECT_HIGH_PITCH,
 	EFFECT_NO_SKY,
 	EFFECT_PLAYER_GTA_2,
+    EFFECT_PEDS_TPOSE,
```
Comment (Last0xygen) [inline ChaosMod/Effects/EffectsInfo.h:260]: these should not be a new line, please merge the master into your branch

File: ChaosMod/Effects/db/Peds/PedsSpeechController.cpp [hunk 2]
```diff
@@ -46,4 +46,27 @@ static void OnTickUnfriendly()
 	}
 }
 
-static RegisterEffect registerEffect2(EFFECT_PEDS_INSULT, nullptr, nullptr, OnTickUnfriendly);
\ No newline at end of file
+static RegisterEffect registerEffect2(EFFECT_PEDS_KIFFLOM, nullptr, nullptr, OnTickUnfriendly);
```
Comment (Last0xygen) [inline ChaosMod/Effects/db/Peds/PedsSpeechController.cpp:49]: this should still be "EFFECT_PEDS_INSULT"

**Review by Last0xygen — CHANGES_REQUESTED** [review]

File: ChaosMod/Effects/db/Peds/PedsSpeechController.cpp (hunk 2 above)
Comment (EliRickard) [reply ChaosMod/Effects/db/Peds/PedsSpeechController.cpp:49]: oh yeah true, late night in visual makes me do weird things. I'll fix it in the morning

File: ChaosMod/Effects/db/Peds/PedsSpeechController.cpp [hunk 3]
```diff
@@ -46,4 +46,27 @@ static void OnTickUnfriendly()
...
+	{
+		lastTick = curTick;
+
+		for (Ped ped : GetAllPeds())
+		{
+			if (!IS_PED_A_PLAYER(ped) && IS_PED_HUMAN(ped))
+			{
+				constexpr const char* speechesKifflom{ "KIFFLOM_GREET" };
+
+				_PLAY_AMBIENT_SPEECH1(ped, speechesKifflom, "SPEECH_PARAMS_FORCE_SHOUTED", 1);
```
Comment (kolyaventuri) [inline ChaosMod/Effects/db/Peds/PedsSpeechController.cpp:66]: The `constexpr` here feels unnecessary. Can you not just pass in `"KIFFLOM_GREET"` in place of `speechesKifflom` as the `speechName` param, given there's only one option?

**Comment by pongo1231** [comment]
Fixed it for you

**Pull request merged**